### DISCUSSION TITLE
How can I modify the click event of the download button?

### DISCUSSION THREAD

**Discussion by andyxiong11** [mozilla/pdf.js#19143] [Q&A] [answered]
I need to modify the response event for clicking the download button to customize the download method. Please tell me where it is located in the viewer.js file
![image](https://github.com/user-attachments/assets/fac2ea92-b840-4aba-878b-e8b32915e8c0)

**Comment by nicolo-ribaudo** [comment, marked as answer]
https://github.com/mozilla/pdf.js/blob/97c7a8eb7a44a887dbbca408c3f0563efea01c7b/web/app.js#L1921
